test/compilable/test22928.c

```diff
@@ -0,0 +1,8 @@
+// https://issues.dlang.org/show_bug.cgi?id=22928
+
+void fn()
+{ 
```

Comment:
the whitespace is here, after the `{`

Reply:
I see, thanks. How awful the error messages are, with no line number. Sigh.

Reply:
I can check and try to address that

Reply:
Btw, you can run pre-commit locally and it will fix this issues automatically, before doing a commit. This is intended to fix the problem and create unstaged changes in files in-place. I will create a PR in the CI config to add a diff.